be/src/exec/vectorized/cross_join_node.cpp

```diff
@@ -51,6 +53,25 @@ Status CrossJoinNode::prepare(RuntimeState* state) {
     _probe_rows_counter = ADD_COUNTER(runtime_profile(), "ProbeRows", TUnit::UNIT);
 
     _init_row_desc();
+    if (_parent != nullptr) {
```

Comment:
ObjectColumn appears in left side is very dangerous,  while in the right side is safe. so follow codes can not work  expectedly.  and some comment. 

change 
```cpp
   if (_parent != nullptr) {
        if (_parent->type() == TPlanNodeType::PROJECT_NODE) {
            const auto& slot_ids = reinterpret_cast<ProjectNode*>(_parent)->slot_ids();
            for (size_t i = _probe_column_count; i < _build_column_count + _probe_column_count; i++) {
                if (_col_types[i]->type().type == PrimitiveType::TYPE_OBJECT) {
                    _deep = false;
                    for (size_t j = 0; j < slot_ids.size(); j++) {
                        if (slot_ids[j] == _col_types[i]->id()) {
                            _deep = true;
                            break;
                        }
                    }
                    if (_deep) {
                        break;
                    }
                }
            }
        }
    }
```
to 
```cpp
            _deep = false;
            for (size_t i = 0; i <  _probe_column_count; i++) {
                if (_col_types[i]->type().type == PrimitiveType::TYPE_OBJECT) {
                    _deep = true;
                    break;
                }
            }
```